packages/core/src/components/table/table-component-expandable-rows.stories.tsx

```diff
@@ -131,20 +131,41 @@ export default {
         type: 'radio',
       },
       options: ['auto', 'hidden'],
+      autoCollapse: {
+        name: 'Auto Collapse',
+        description: 'Automatically collapses other rows when one is expanded.',
+        control: {
+          type: 'boolean',
+        },
+        table: {
+          defaultValue: { summary: false },
+        },
+      },
+    },
+    args: {
+      modeVariant: 'Inherit from parent',
+      compactDesign: false,
+      expanded: false,
+      responsiveDesign: false,
+      verticalDivider: false,
+      noMinWidth: false,
+      column1Width: '',
+      column2Width: '',
+      column3Width: '',
+      column4Width: '',
+      overflow: 'scroll',
+      autoCollapse: false,
+    },
+    autoCollapse: {
```

Comment:
Seems like there might be a duplicate of this Storybook control, it takes in "args" as a control and doesn't set a default value for the "autoCollapse" control
<img width="1453" alt="Screenshot 2024-10-11 at 09 24 02" src="https://github.com/user-attachments/assets/c867454b-43b9-4f58-8b1b-0022e07598b1">